src/main/resources/db/migration/V3_6__add_errand_id_to_message_and_attachment_tables.sql

```diff
@@ -0,0 +1,5 @@
+alter table if exists message add column if not exists errand_id bigint;
```

Comment:
File is named` V3_6__add_errand_id_to_message_and_attachment_tables.sql`but unless I'm misstaken only message is updated?

Reply:
Yes, but we adds errand_id:s to both table (both structure/data in message and only data in attachment)